Formula/l/llgo.rb

```diff
@@ -20,8 +19,10 @@ class Llgo < Formula
     sha256               x86_64_linux:  "10a922642cecc66b75420e9d661e4c12409c2ed7d379123c0d52d339b86de938"
   end
 
+  depends_on "go" => :build
```

Comment:
If `go` is still also a runtime dependency, it does not need to be specified here for build.

Reply:
Updated